md-dao/src/main/java/com/wipro/ats/bdre/md/dao/DeployDAO.java

```diff
@@ -60,30 +61,30 @@ public void initDeploy(Long deployId) {
             pickedDeployStatus.setDeployStatusId((short) 5);
 
             // querying deploymentId present or not
-            Criteria checkDeploymentId = session.createCriteria(ProcessDeploymentQueue.class).add(Restrictions.eq("deploymentId", deployId));
+            Criteria checkDeploymentId = session.createCriteria(ProcessDeploymentQueue.class).add(Restrictions.eq(DEPLOYMENTID, deployId));
             Integer deploymentIdCount = checkDeploymentId.list().size();
             LOGGER.info("Deployment Id count :" + deploymentIdCount);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Deployment Id count :" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)